flow/scripts/gui.tcl

```diff
@@ -47,8 +47,11 @@ if {![info exist ::env(GUI_NO_TIMING)]} {
   if {$design_stage >= 6 && [file exist $::env(RESULTS_DIR)/6_final.spef]} {
     puts "Loading spef"
     read_spef $::env(RESULTS_DIR)/6_final.spef
+  } elseif {$design_stage >= 5} {
+    puts "Estimating parasitics -global_routing"
+    estimate_parasitics -global_routing
```

Comment:
This is inconsistent with the logging that I've been adding lately. Time to make a utility fn?

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/8abd546de390d055f2f7e54483cd3520a472773c/flow/scripts/cts.tcl#L36

Reply:
How should it read?  What would the utility do?

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2007